pandas/core/frame.py

```diff
@@ -7939,8 +7947,8 @@ def idxmin(self, axis=0, skipna=True):
         Parameters
         ----------
         axis : {0 or 'index', 1 or 'columns'}, default 0
-            0 or 'index' for row-wise, 1 or 'columns' for column-wise
-        skipna : bool, default True
+            The axis to use. 0 or 'index' for row-wise, 1 or 'columns' for column-wise
+        skipna : boolean, default True
```

Comment:
I think this should stay `bool` - surprised it doesn't error on validation

Reply:
I'm actually surprised to see this in here at all, as I was only editing the `_misc.py` file? still figuring out how to merge fork/merge/etc correctly

Reply:
```sh
git fetch upstream
git merge upstream/master
# fix merge conflicts, if any then optionally
# git merge continue
```

Reply:
got it, thanks. i think also having my local branch have the upstream as `upstream/master` was causing problems.